### PR TITLE
Resolve feature Image Scan

### LINKED ISSUE #1: Image Scan
Front end: Create UI for image processing
Scan images live/upload images -> show results from the backend
Integrate camera, and buttons like that

### PR DESCRIPTION
**Pull request by thuyvy119**
Close #1 
- Added ImageScan component for ingredients page (include capture, retake and submit functions).

https://github.com/hungngodev/Flavorie/assets/45311428/d5c6fb29-841a-4e43-a081-360d570f303a

### CODE REVIEW

File: client/src/components/ingredients/ImageScan.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    const retake = () => {
+        setImgSrc(null);
+        setMessage(null);
+    };
+
+    // submit photo
+    const submit = () => {
+        if (imgSrc) {
+            setMessage('Sucessfully submitted');
+        }
```
Comment (hungngodev) [inline client/src/components/ingredients/ImageScan.tsx:29]: Please add some comments add here to let NA know that the image will be submitted at here!

File: client/src/components/ingredients/ImageScan.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    };
+
+    return (
+        <Flex className="container" direction='column' align='center' justify='center'>
+            {imgSrc ? (
+                <img src={imgSrc} alt="webcam" />
```
Comment (hungngodev) [inline client/src/components/ingredients/ImageScan.tsx]: Also please use the Image component of charka UI. ITs width and height should be the same as the Webcam components.

**Review by hungngodev — CHANGES_REQUESTED** [review]

File: client/src/components/ingredients/ImageScan.tsx (hunk 2 above)
Comment (thuyvy119) [reply client/src/components/ingredients/ImageScan.tsx]: Done.

File: client/src/components/ingredients/ImageScan.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    const submit = () => {
+        if (imgSrc) {
+            setMessage('Sucessfully submitted');
+        }
+    };
+
+    return (
+        <Flex className="container" direction='column' align='center' justify='center'>
+            {imgSrc ? (
+                <Image src={imgSrc} alt="webcam"/>
```
Comment (hungngodev) [inline client/src/components/ingredients/ImageScan.tsx:35]: Please add width and height for it!

**Review by hungngodev — CHANGES_REQUESTED** [review]
Please also fix the merge conflict after resolving all pending request changes!

File: client/src/components/ingredients/ImageScan.tsx (hunk 3 above)
Comment (thuyvy119) [reply client/src/components/ingredients/ImageScan.tsx:35]: Resolved conflict.

**Review by hungngodev — APPROVED** [review]
Thank you for merging the conflicts!

**Comment by hoatuyet423** [comment]
Hmm, why is this in open PR ??? lol

**Pull request merged**